plugins/docker/docker_

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+"""
+=head1 NAME
+
+docker_ - Docker wildcard-plugin to monitor a L<Docker|https://www.docker.com> host.
+
+This wildcard plugin provides at the moment only the suffixes C<containers>, C<images>, C<status>,
+C<volumes>, C<cpu> and C<memory>.
+
+=head1 INSTALLATION
+
+- Copy this plugin in your munin plugins directory
+- Install Python3 "docker" package
+
+=over 2
+
+    chmod 755 docker_
+
+    ln -s /usr/share/munin/plugins/docker_ /etc/munin/plugins/docker_containers
+    ln -s /usr/share/munin/plugins/docker_ /etc/munin/plugins/docker_cpu
+    ln -s /usr/share/munin/plugins/docker_ /etc/munin/plugins/docker_images
+    ln -s /usr/share/munin/plugins/docker_ /etc/munin/plugins/docker_memory
+    ln -s /usr/share/munin/plugins/docker_ /etc/munin/plugins/docker_status
+    ln -s /usr/share/munin/plugins/docker_ /etc/munin/plugins/docker_volumes
+
+=back
+
+After the installation you need to restart your munin-node:
+
+=over 2
+
+    systemctl restart munin-node
+
+=back
+
+=head1 CONFIGURATION
+
+When you want to change the default host (unix://var/run/docker.sock) or any other things
+(see https://docs.docker.com/compose/reference/envvars/) do it in a file named docker placed
+in the directory /etc/munin/plugin-conf.d/ with a config like this:
+
+=over 2
+
+    [docker_*]
+    user root
```

Comment:
I guess, the `user root` setting is *always* necessary in order to gain access to the control socket?

Reply:
Yes, I wrote a precision for this in documentation.